tests/installation/live_installation.pm

```diff
@@ -48,7 +49,14 @@ sub run {
         assert_and_click 'live-upgrade';
     }
     else {
-        x11_start_program('xdg-su -c "/usr/sbin/start-install.sh"', target_match => 'maximize');
+        if (is_aarch64) {
+            # On aarch64 there is sporadic issue when "Install" icon is clicked too long,
+            # so that context menu is called instead of opening the wizard.
+            x11_start_program('xdg-su -c "/usr/sbin/start-install.sh"', target_match => 'maximize');
```

Comment:
should we display a softfail here to know that this is not the expected user experience?

Reply:
There is no ticket for this, so I cannot add soft-fail here. And I'm not sure if we need ticket for this at all, as it does not look like something we can fix.